plugins/modules/azure_rm_keyvault.py

```diff
@@ -273,6 +273,13 @@
                         description:
                             - Property to specify whether NRP will ignore the check if parent subnet has serviceEndpoints configured.
                         type: bool
+    is_purge_deleted:
+        description:
+            - Whether permanently deletes the specified vault. aka Purges the deleted Azure key vault
+            - Default value is C(True)
+            - When I(is_purge_deleted) need I(location) to be configured, If not configured, the default locatioin of the resource group.
```

Comment:
```suggestion
            - When I(is_purge_deleted) is specified, the I(location) has to be configured. If not configured, the default locatioin of the resource group will be used.
```

BTW, isn't the `location` always available for this resource?

Reply:
No , the location required for purge

Reply:
Look for the exist name within  the location